daemon/queue/NzbFile.cpp

```diff
@@ -391,15 +391,31 @@ void NzbFile::ProcessFiles()
 
 	if (m_password)
 	{
-		ReadPassword();
+		ReadPasswordFromNzb();
+	}
+	ReadPasswordFromFilename();
```

Comment:
If `ReadPasswordFromNzb` finds the password do we need to look for another password in the NZB filename?
Shouldn't the logic be like
```
if (m_password)
{
   ReadPasswordFromNzb();
}
else
{
   ReadPasswordFromFilename();
}
```

Reply:
I have updated the i have updated the logic in the pull request to reflect that change. I wasnt entirely sure what the possible states of the var m_password is, or how the rest of the program responds to it. So far my manual test work as well, so if you think that logic is better, lets go with it.